src/app/bin/app.py

```diff
@@ -2,6 +2,8 @@
 
 import sys
 from os import path
+import warnings
+warnings.filterwarnings("ignore", message= 'calling yaml*')
```

Comment:
I added the module and changed the message, but I don't think YAMLLoadWarning is an actual category warnings would recognize. I did try setting it as a deprecation warning but it doesn't seem to register as that.

`warnings.filterwarnings("ignore", module="apispec.utils", message= "calling yaml")`

Reply:
> I added the module and changed the message, but I don't think YAMLLoadWarning is an actual category warnings would recognize. I did try setting it as a deprecation warning but it doesn't seem to register as that.
> 
> So it would now look like
> 
> `warnings.filterwarnings("ignore", module="apispec.utils", message= "calling yaml.Load()")`

Documentation for what a category is can be found [here](https://docs.python.org/3/library/warnings.html#the-warnings-filter).  Basically it has to be a subsclass of `Warning`, which YAMLLoadWarning is:

```python
In [1]: from yaml import YAMLLoadWarning

In [2]: issubclass(YAMLLoadWarning, Warning)
Out[2]: True
```

Having said that, doing the combination of module and message like you did is also acceptable.  Let's make the message "calling yaml.load".  Again, the regex thing:
* The message regex gets compiled as case insensitive, so `Load` vs `load` won't matter in that regard, but the actual message is lowercase `load`.
* The `()` in regex won't be interpreted literally, but is used to capture what's inside the parenthesis (nothing in this case) for later use.  So just leaving off the `()` is functionally equivalent in this case and makes what we're checking seem more intentional.